spacy/pipeline/entity_linker.py

```diff
@@ -67,7 +67,7 @@ def make_entity_linker(
     incl_prior: bool,
     incl_context: bool,
     entity_vector_length: int,
-    get_candidates: Callable[[KnowledgeBase, "Span"], Iterable[Candidate]],
+    get_candidates: Callable[[KnowledgeBase, Span], Iterable[Candidate]],
```

Comment:
@svlandeg: There's no reason to keep this as a forward ref, is there?

Reply:
No, this edit should be fine!